source/govuk-payment-pages.html.erb

```diff
@@ -65,9 +65,10 @@ title: GOV.UK payment pages
         <li>give full or partial refunds</li>
         <li>report on payments manually with CSV downloads or automatically via our API</li>
         <li>apply custom branding on payment pages</li>
+        <li>take payments by phone or post</li>
```

Comment:
I think this would be better worded as "allow your staff to process payments taken over the phone or by post"